src/api/vdom.js

```diff
@@ -153,7 +152,7 @@ function elementOpenStart(...args) {
 }
 
 function elementOpenEnd() {
-  const node = newElementOpen(...overrideArgs);
+  const node = newElementOpen(...overrideArgs); // eslint-disable-line no-use-before-define
```

Comment:
It could be complaining due to function hoisting, in which case doing `const elementOpenEnd = ...` should fix it. But I feel like we're having to jump through unnecessary hoops just to satisfy an overexcited linting rule.